lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -602,3 +603,14 @@ func getNetworkProbeConfig(networkID string) ([]*lte_mconfig.NProbeTask, *lte_mc
 	}
 	return npTasks, liUes
 }
+
+// getSyncInterval takes network-wide subscriber_db sync interval and overrides it if also set for gateway
+func getSyncInterval(nwEpc *lte_models.NetworkEpcConfigs, gwEpc *lte_models.GatewayEpcConfigs) uint32 {
+	sync_interval := nwEpc.SubscriberdbSyncInterval
+	fmt.Printf("nw: %d\n", nwEpc.SubscriberdbSyncInterval)
```

Comment:
If these are just for testing, let's remove them. Otherwise, we use `glog` for logging.

Reply:
oh my.. Thanks for catching these. Unintentional.